src/components/admin/Election.js

```diff
@@ -14,6 +14,14 @@ export default function Election() {
   return (
     <div>
       <h1>Nueva Eleccion</h1>
+      <a
+        className="App-link"
+        href=".netlify/functions/hello"
```

Comment:
Al ejecutarlo la ruta queda asi: http://localhost:8888/admin/.netlify/functions/hello
```suggestion
        href="../.netlify/functions/hello"
```

Reply:
Sí, gracias! Igual creo que no voy a necesitar hacer merge de este PR. Simplemente lo estoy usando para probar la base de datos que configuré en Mongo Atlas.